scripts/send-message-from-millau-rialto.sh

```diff
@@ -15,20 +15,14 @@ case "$1" in
 			--source-host localhost \
 			--source-port $MILLAU_PORT \
 			--source-signer //Alice \
-			--target-signer //Bob \
-			--lane 00000000 \
-			--origin Target \
-			remark \
+			raw 020419ac
```

Comment:
@svyatonik 
Slava, how was this value `020419ac` generated? 
I can see, that it is kind of "trap message for Kusama",

we also should fix this send-message script,
do we want to leave here remark/transfer? how to fix them?
or what do you suggest?


https://substrate.stackexchange.com/questions/5901/failed-to-run-parity-bridges-common-message-realy-demo/5906?noredirect=1#comment5781_5906

Reply:
IIRC it is just a `Trap(42)` - just to see that it is dispatched in logs (instead of some "Unable to decode XCM"). Send message script is in abandoned state currently (as well as its complicated version - https://github.com/svyatonik/parity-bridges-common.test) - imo we'll need fix it after stabilizing everything + updating project-level documentation.

Reply:
@svyatonik Slava, so you dont remember how was that `Trap(42)` encoded to `020419ac`?

Reply:
Not sure, but probably I've just printed it in [this test](https://github.com/paritytech/parity-bridges-common/blob/master/bin/millau/runtime/src/xcm_config.rs#L302).